cloc/utils.py

```
@@ -0,0 +1,314 @@
+"""
+Utility functions for CLOC tool.
+"""
+
+import os
+import re
+from pathlib import Path
+from typing import Dict, List, Set, Tuple, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)